packages/modules/data-widgets/src/themesource/datawidgets/web/datagrid.scss

```diff
@@ -100,6 +100,7 @@ $brand-primary: #264ae5 !default;
                 margin-left: 8px;
                 flex-grow: 0;
                 color: $icon-color;
+                height: 14px;
```

Comment:
Why we need this space now?

Reply:
There is no font-size being applied to the container so the SVG is presented in the original size. In this way it will use the same size as before.